cmd/headers.go

```diff
@@ -80,6 +80,7 @@ config, see the "copywrite init" command.`,
 		// Append default ignored search patterns (e.g., GitHub Actions workflows)
 		autoSkippedPatterns := []string{
 			".github/workflows/**",
+			"node_modules/**",
```

Comment:
can `node_modules` also exist in subdirs?

Reply:
Ahh that's a good point... With monorepos in play, yes. Most of the projects at HashiCorp appear to use yarn workspaces, which should hoist things into the `node_modules` folder in the root, but that can't be guaranteed.

I'll update to `**/node_modules/**`